(DOCSP-17707): Update code examples and text to better illustrate offline vs online opening a synced realm (#1247)
mongodb/docs-realm: source/sdk/ios/examples/configure-and-open-a-realm.txt

A **{+realm+}** is the core data structure used to organize data in
{+client-database+}. For more information, see :ref:`ios-realms`.

.. seealso::

>    This page describes the options to configure and open a local 

Reviewer: ```suggestion
   This page explains how to configure and open a local 
```
"describes the options to" sounds not-specific and wordy to me -- suggesting this reword. Take it or leave it, this is probably just subjective instinct on my part.

   {+client-database+}. For information about opening a synced {+realm+},

>    including how to open a synced realm offline or online-only, see:

Reviewer: ```suggestion
   including how to open a synced realm offline or online-only, see
```
Usually I don't bother with a colon when introducing a single page in a note like this, so I know there are a lot of examples of me *not* including a colon here. Might be a good idea to keep this consistent with that so different parts of the docs don't have different styles of note.
